_includes/head.html

```diff
@@ -31,4 +31,5 @@
   </style>
 <![endif]-->
 
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<meta http-equiv="cleartype" content="on">
+<script async defer src="{{ '/assets/js/main.min.js' | absolute_url }}"></script>
```

Comment:
It provides [better fallback support](http://stackoverflow.com/a/13821233) for IE. I picked up the pattern from the CloudCannon team. IMO we can ((and always should) expect browsers to make the appropriate choice when both are provided.